packages/expandable/src/props.tsx

```diff
@@ -32,6 +32,12 @@ export type ExpandableProps = {
    */
   expanded?: boolean;
 
+  /**
+   * The state of the chevron-up icon, if set to true, chevron up should be displayed, if set to false then chevron-down icon should be displayed.
+   * @default false
+   */
+  showChevronUpIcon?: boolean;
```

Comment:
Why can't we use `expanded` for that purpose? If it's `true`, chevron down should be displayed and the opposite.